webapp-django/crashstats/topcrashers/tests/test_views.py

```diff
@@ -41,15 +42,15 @@ def mocked_signature_first_date_get(**options):
                         "signature": u"FakeSignature1 \u7684 Japanese",
                         "first_date": datetime.datetime(
                             2000, 1, 1, 12, 23, 34,
-                            tzinfo=UTC(),
+                            tzinfo=utc,
                         ),
                         "first_build": "20000101122334",
                     },
                     {
                         "signature": u"mozCool()",
                         "first_date": datetime.datetime(
                             2016, 5, 2, 0, 0, 0,
-                            tzinfo=UTC(),
+                            tzinfo=utc,
```

Comment:
Mmm, sorry, I made these changes while I was working on the ``get_date_boundaries`` function, but then I ended up moving it somewhere else, so it's weird that this file has any changes. Luckily, it's all very simple.